src/ml_tooling/transformers.py

```diff
@@ -39,9 +39,17 @@ class FillNA(BaseEstimator, TransformerMixin):
     Fills NA values with given value or strategy. Either a value or a strategy has to be supplied.
     """
 
-    def __init__(self, value: int = None, strategy: str = None):
-        self.value = value
+    def __init__(self, value=None, strategy: str = None):
+
+        if value is None and strategy is None:
```

Comment:
Is there any preference in ml_tooling to when to check input? At init or fit? @andersbogsnes

Reply:
The pythonic way is to fail fast - If we can fail at __init__, thats the best place to fail

Reply:
Noted :-)